modules/applications/pages/resources.adoc

```diff
@@ -1,7 +1,21 @@
 = Resources
-:description:
+:description: Here, you'll find a curated collection of references to help you maximize your productivity and enhance your understanding of Bonita UI Builder. 
 
-[NOTE]
-====
-For Subscription editions only.
-====
+{description}
+
+
+== Bonita UI Builder documentation
+* xref:applications:initiation-manual.adoc[Initiation manual] to help you start from scratch and build your first application 
+* xref:applications:how-tos-builder.adoc[Guides] to facilitate your usage of Bonita UI Builder
+* xref:applications:faq.adoc[FAQ]
+
+ 
+
+== Other documentations
+* https://api-documentation.bonitasoft.com/latest/[Bonita API] : designed to help you interact with your Bonita projects and data from Bonita UI Builder
```

Comment:
Don't use url in documentation but use variable define in antora.yml (on the root of this project)
```suggestion
* {openApiUrl}/{openApiLatestVersion}[Bonita API] : designed to help you interact with your Bonita projects and data from Bonita UI Builder
```